### DISCUSSION TITLE
I dont understand allowed_mesh_ids

### DISCUSSION THREAD

**Discussion by uwecash** [fsaris/home-assistant-awox#120] [General]
Hi! Great job first of all! I have 2 Esp running for two Awox LED controllers. I also have to place each Esp very close to the respective LED controller. One person in particular loses the connection very quickly, but that was already the case with the Awox app. I then assigned the respective Mac address to each Esp. According to the log, this worked for a few hours. Unfortunately I no longer have the logs. I then wanted to add the allowed_mesh_ids option into the config, but I don't understand the values. Where do I get the value for the respective device?

**Comment by fsaris** [comment]
The mesh IDs are mentioned in the logs, but also part of the mqtt message. If you use home assistant the mesh ID is part the device name, the value between the brackets.
